app/models/email_history.rb

```diff
@@ -7,6 +7,10 @@ class EmailHistory < ApplicationRecord
     SUITABLE_FOR_EARLY_ALLOCATION = 'suitable_for_early_allocation',
     OPEN_PRISON_COMMUNITY_ALLOCATION = 'open_prison_community_allocation',
     IMMEDIATE_COMMUNITY_ALLOCATION = 'immediate_community_allocation',
+    RESPONSIBILITY_OVERRIDE = 'responsibility_override',
+    URGENT_PIPELINE_TO_COMMUNITY = 'urgent_pipeline_to_community',
+    ASSIGN_COM_LESS_THAN_10_MONTHS = 'assign_com_less_than_10_months',
+    COMMUNITY_EARLY_ALLOCATION = 'community_early_allocation'
```

Comment:
Unsure about it, but now we've added more events here, do we also need to filter some of them in the `scope :in_offender_timeline`? Not really sure, just mentioning it in case you know more!